force-app/main/default/lwc/apexImperativeMethod/__tests__/apexImperativeMethod.test.js

```diff
@@ -131,10 +122,12 @@ describe('c-apex-imperative-method', () => {
         const buttonEl = element.shadowRoot.querySelector('lightning-button');
         buttonEl.click();
 
-        return Promise.resolve().then(() => expect(element).toBeAccessible());
+        await flushPromises();
+
+        expect(element).toBeAccessible();
```

Comment:
`.toBeAccessible()` returns a promise so you need to put an `await` keyword in front of this statement.
Please apply this to all instances of `.toBeAccessible()`.

Reply:
Changes done!